pycodestyle.py

```diff
@@ -133,7 +133,7 @@ def lru_cache(maxsize=128):  # noqa as it's a fake implementation.
 RAISE_COMMA_REGEX = re.compile(r'raise\s+\w+\s*,')
 RERAISE_COMMA_REGEX = re.compile(r'raise\s+\w+\s*,.*,\s*\w+\s*$')
 ERRORCODE_REGEX = re.compile(r'\b[A-Z]\d{3}\b')
-DOCSTRING_REGEX = re.compile(r'u?r?["\']')
```

Comment:
russian strings (`ru'foo'`) are not valid -- could you also add a test case to demonstrate your change while you're at it -- thanks!


```console
$ python2
Python 2.7.15+ (default, Nov 27 2018, 23:36:35) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> ru'foo'
  File "<stdin>", line 1
    ru'foo'
          ^
SyntaxError: invalid syntax
>>> 
$ python3
Python 3.6.8 (default, Jan 14 2019, 11:02:34) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> ru'foo'
  File "<stdin>", line 1
    ru'foo'
          ^
SyntaxError: invalid syntax
```

Reply:
@xuhdev I'm looking at the PR diff here and am not seeing a test or change that matches what @asottile asked for